### PR TITLE
selectfasta

### PR DESCRIPTION
**Pull request by andvides**
**Replace this sentence with a detailed description of your pull request**

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [y ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ y] This PR adds a new recipe.
* [ y] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: recipes/selectfasta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+test:
+  commands:
+    - selectFasta -h
+
+about:
+  home: https://github.com/andvides/selectFasta/
+  license: GNU
```
Comment (npavlovikj) [inline recipes/selectfasta/meta.yaml]: You will need to add proper license file/information in the repo.

Comment (andvides) [reply recipes/selectfasta/meta.yaml]: done

**Comment by npavlovikj** [comment]
The new error message is:
```
14:32:49 BIOCONDA INFO (OUT)   File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 763, in copy_license
14:32:49 BIOCONDA INFO (OUT)     "source root dir or in recipe root dir (with meta.yaml)".format(src_file))
14:32:49 BIOCONDA INFO (OUT) ValueError: License file given in about/license_file (/opt/recipe/LICENSE) does not exist in source root dir or in recipe root dir (with meta.yaml)
```
Please make sure the License file is in the tarball and the naming is correct. I would suggest you to add it in the tarball if it is not there since you are the developer on the package.

Moreover, while the build passed on Linux yesterday, it failed on MacOS. I would suggest you to take a look at that and see if you can fix that error.

**Comment by andvides** [comment]
Done

**Comment by andvides** [comment]
License updated

**Comment by npavlovikj** [comment]
Thanks @andvides !

As you can see, the OSX build is failing, https://circleci.com/gh/bioconda/bioconda-recipes/99279?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link.
All the builds should pass and everything should be green before we merge the recipe.
Were you be able to fix the OSX build according to your experience with the package and the error message?

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Comment by npavlovikj** [comment]
@BiocondaBot please merge

**Comment by npavlovikj** [comment]
Thank you for contributing @andvides !

**Pull request merged**